app.getspace.Space.json

```diff
@@ -1,15 +1,17 @@
 {
   "app-id": "app.getspace.Space",
   "runtime": "org.freedesktop.Platform",
-  "runtime-version": "20.08",
+  "runtime-version": "24.08",
   "sdk": "org.freedesktop.Sdk",
   "command": "space",
   "separate-locales": false,
   "finish-args": [
     "--share=network",
     "--socket=wayland",
+    "--share=ipc",
     "--socket=fallback-x11",
-    "--device=dri"
+    "--device=dri",
+    "--persist=."
```

Comment:
why persist `.` ?

Reply:
The application stores some files and a directory. Downloaded content and user progress are stored.

Reply:
graphql.hive, graphql.lock and the libCachedImageData directory are stored. Without setting the permission, the work is not preserved.

Reply:
what are the *exact path* of each?

Reply:
So `graphql.hive` and `graphql.lock` are **files** in the home directory?

Reply:
yes